components/illustratedmessage/stories/illustratedmessage.test.js

```diff
@@ -10,16 +10,27 @@ export const IllustratedMessageGroup = Variants({
 		description: [
 			"This page isn't available. Try checking the URL or visit a different page.",
 		],
-		useAccentColor: false,
-	}, {
-		testHeading: "With accent color",
+	},
+	{
+		testHeading: "Horizontal layout",
+		isHorizontal: true
+	},
+	{
+		testHeading: "With link",
 		heading: "Drag and drop your file",
 		description: [
 			() => {
 				return html`${Link({ url: "#", text: "Select a file" })} from your computer.`;
 			},
 		],
-		useAccentColor: true,
+	},
+	{
+		testHeading: "W/o button group",
+		hasButtons: false
+	},
+	{
+		Template: SizingTemplate
 	}
 	],
+	withSizes: false
```

Comment:
Is there a reason that the `SizingTemplate` is used instead of `withSizes`?

As a side note, it'd also might be nice to show the horizontal variant in all sizes, but I can also see the argument for not doing that.